salvo-remote/salvo_remote.go

```diff
@@ -1,9 +1,18 @@
+// salvo-remote instruments the execution of performance tests on a sandbox
+// running remotely on AWS.
 package main
 
-import "fmt"
-import "os"
+import (
+	"flag"
+	"fmt"
+	"os"
+)
+
+var buildID = flag.String("build_id", "", "The ID of the AZP build that produced components and binaries for this salvo-remote execution. Can be overridden by providing a different ID in -build_id_override.")
```

Comment:
Please break it to multiple lines as usual.

Reply:
Unlike other languages, we don't observe a specific line length limit in Go. We still try to keep lines short by refactoring (e.g. a function that takes too many arguments), but we don't break long strings or similar. Breaking a long string complicates readability, since newline characters or plus signs are added for no benefit, since most modern text editors can visually wrap the line for the reader without the added complications.

https://google.github.io/styleguide/go/guide#line-length